src/components/Counter.jsx

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+
+const Counter = () => {
+    const[count, setCount] = useState(0)
+    function increment(){
```

Comment:
tip: these are called handler functions that usually names starts with `handle` like `handleIncrement` and so on.

Reply:
- Rename the function increment to handleIncrement
- Rename the function decrement to handleDecrement